Limb Darkening RtD

catherine-martlin opened the pull request:
Updated limb_darkening tutorial for read the docs

Commit:
Updated limb_darkening tutorial for read the docs

Commit:
Small updates for rending

nespinoza commented:
Need to update env + pip_reqs files before we can merge. @mfixstsci mentioned he can do this.

taylorbell57 committed:
Fix ModelGrid customize table filtering

Build the parameter mask separately before applying it to the Astropy table,
avoiding nested indexing that fails on newer Astropy versions.

taylorbell57 committed:
Updating exoctk.contam_visibility.rst submodule list

taylorbell57 committed:
Add conversion for effective wavelengths in LDC class

taylorbell57 committed:
Fixing UserWarning about positioning legend location before legend declaration

taylorbell57 committed:
Adding esbonio for docs optional dependency group for local preview of RST files

taylorbell57 committed:
Updating LimbDarkening tutorial with updates from the psat 6 years

taylorbell57 committed:
Adding documentation about SPAM coefficients

hover2pi reviewed (approved):
Looks great! Thanks @catherine-martlin (Sorry this took sooooo long! Holy smokes.) and @taylorbell57 !
